applications/NXstress.nxdl.xml

```diff
@@ -0,0 +1,1115 @@
+                    <attribute name="interpretation" type="NX_CHAR">
+                        <enumeration>
+                            <item value="spectrum" />
+                        </enumeration>
+                    </attribute>
+                    <attribute name="units" type="NX_CHAR">
+                        <doc>Specify the *y_Unit* units</doc>
+                    </attribute>
+                </field>
+
+                <field name="fit" type="NX_NUMBER">
+                    <doc>Diffractogram fit counts (auxiliary signal).</doc>
+                    <dimensions rank="2">
+                        <dim index="1" value="n_D" />
+                        <dim index="2" value="n_X" />
+                    </dimensions>
+                    <attribute name="interpretation" type="NX_CHAR">
+                        <enumeration>
+                            <item value="spectrum" />
+                        </enumeration>
+                    </attribute>
+                </field>
+
+                <field name="fit_errors" type="NX_NUMBER">
+                    <doc>Diffractogram fit counts error (auxiliary signal).</doc>
+                    <dimensions rank="2">
+                        <dim index="1" value="n_D" />
+                        <dim index="2" value="n_X" />
+                    </dimensions>
+                </field>
+
+                <field name="background" type="NX_NUMBER" minOccurs="0" maxOccurs="1">
+                    <doc>In case the diffraction background was manually determined. Diffractogram background counts (auxiliary signal).</doc>
+                    <dimensions rank="2">
+                        <dim index="1" value="n_D" />
+                        <dim index="2" value="n_X" />
+                    </dimensions>
+                    <attribute name="interpretation" type="NX_CHAR">
+                        <enumeration>
+                            <item value="spectrum" />
+                        </enumeration>
+                    </attribute>
+                </field>
+                <field name="residuals" type="NX_NUMBER" minOccurs="0" maxOccurs="1">
+                    <doc>Difference between diffractogram and fit (auxiliary signal).</doc>
+                    <dimensions rank="2">
+                        <dim index="1" value="n_D" />
+                        <dim index="2" value="n_X" />
+                    </dimensions>
+                    <attribute name="interpretation" type="NX_CHAR">
+                        <enumeration>
+                            <item value="spectrum" />
+                        </enumeration>
+                    </attribute>
+                </field>
+            </group>
+        </group>
+
+        <group name="NOTES" nameType="any" type="NXnote" minOccurs="0" maxOccurs="1">
+            <doc>
+                User description of the data acquisitions.
+                A description of data analysis goes in the
+                *fig_log group*.

```

Comment:
Where is this?

Reply:
This must have been its original name. It is now /NXstress/ENTRY/FIT/description group.

Reply:
I see that /NXstress/ENTRY/FIT/description has this in the docstring "The name will be numbered to allow for ordering of steps."

Note sure what this means. There is only one description.

Reply:
Ah, they forgot nameType="any"